flow/Makefile

```diff
@@ -190,13 +190,13 @@ endif
 # Priority is given to
 #       1 user include path to the executable prior to running make
 #       2 user explicity set with variable in Makefile or command line
-OPENROAD_CMD            ?= $(shell command -v openroad)
-ifeq ($(OPENROAD_CMD),)
-  OPENROAD_CMD           = $(abspath $(FLOW_HOME)/../tools/build/OpenROAD/src/openroad) -no_init -exit
-else
-  OPENROAD_CMD           = $(shell command -v openroad) -no_init -exit
+OPENROAD_EXE            ?= $(shell command -v openroad)
+ifeq ($(OPENROAD_EXE),)
+  OPENROAD_EXE           = $(abspath $(FLOW_HOME)/../tools/build/OpenROAD/src/openroad)
 endif
-OPENROAD_NO_EXIT_CMD     = $(firstword $(OPENROAD_CMD))
+OPENROAD_ARGS           := -no_init $(OPENROAD_ARGS)
```

Comment:
This probably does not work as intended. When setting a variable from the environment, e.g. running `make OPENROAD_ARGS=blah`, make will override any assignments to the variable. So in that example `OPENROAD_ARGS` would be `blah` instead of `-no_init blah`

The only options are to either run `make OPENROAD_ARGS='-no_init blah'` or to make a temp variable, e.g. `OPENROAD_ARGS = -no_init $(OR_ARGS)`, and then run with `make OR_ARGS=blah`. I'm fine with either solution.